COMPATIBILITY.md

```diff
@@ -25,6 +25,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `Fingerprinter` (core)
 - [X] `JUnitResultArchiver` (`junit`)
 - [X] `JavadocArchiver` (`javadoc`)
+- [X] `TapPublisher` (`tap`): [JENKINS-34000](https://issues.jenkins-ci.org/browse/JENKINS-34000) as of 2.0
```

Comment:
Right, sounds like it is 2.1 which actually works.

And for implemented support, the JIRA issue should be omitted. That is only of historical interest and can be obtained where needed from plugin changelogs and/or Git history.